webcompat/static/js/lib/fonts.js

```diff
@@ -0,0 +1,27 @@
+if (/Mobi/.test(navigator.userAgent)) {
```

Comment:
I'm generally not a fan of this kind of UA sniffing... :/

Reply:
Oh... :/
We'd have to use some kind of sniffing to find out if the user in on mobile. Do you think that's a bad idea?
I could remove it and make it run on Desktop & mobile.